src/Controller/Api/SubscriptionController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace AppBundle\Controller\Api;
+
+use AppBundle\Subscription\SubscriptionHandler;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @Route("/email-subscriptions")
+ */
+class SubscriptionController extends Controller
+{
+    /**
+     * @Route("/change", name="app_change_email_subscriptions_webhook")
+     * @Method("POST")
+     */
+    public function changeEmailSubscriptionsAction(Request $request, SubscriptionHandler $subscriptionHandler): Response
```

Comment:
Please either declare this handler as public, or extends the `AbstractController` and declare it in `getSubscribedServices()`.

Reply:
Fixed.